docs_src/V3TopLevelMigration.md

```diff
@@ -102,11 +107,27 @@ The field that has changed in V3 is the `apiVersion` which is now set to `v3`.
 
 Most of the data stored by Core Metadata will be recreated when the V3 versions of the Device Services start-up. The statically declared devices will automatically be created and device discovery will find and add existing devices. Any device profiles, devices, provision watchers created manually via the V2 REST APIs will have to be recreated using the V3 REST API. Any manually-applied `AdministrativeState` settings will also need to be re-applied.
 
+#### API Change
+- Remove `LastConnected`, `LastReported` and `UpdateLastConnected` configuration.
+- Changed database configuration from `Databases map[string]bootstrapConfig.Database` to `Database bootstrapConfig.Database`
+- Added `optional` field in ResourceProperties
+- Updated data type in ResourceProperties
+- Updated ProtocolProperties to have typed value
+- Updated Value Properties max and min data type from `string` to `number`
+- Switched UOM API to YAML format
+- Do not check the empty string profile name when adding the ProvisionWatcher
+
+### Core Command
+#### API Change
+- Updated `ds-pushevent` and `ds-returnevent` to use bool value
+
 ### Support Notifications
 
 Any `Subscriptions` created via the V2 REST API will have to be recreated using the V3 REST API. The `Notification` and `Transmission`collections will be empty until new notifications are sent using EdgeX V3 
 
 ### Support Scheduler
+#### API Change
+- Added authentication to support-scheduler actions
```

Comment:
Add more details about the new `AuthMethod` field
```suggestion
- Added authentication to support-scheduler actions DTO. 
```

Reply:
updated